src/Pages/Register.js

```diff
@@ -1,60 +1,81 @@
 import React, { useState } from 'react';
```

Comment:
so, did you code this page?  I don't think anyone is super protective of the code that's currently in App.js or Dashboard.js but if Edith coded this page, she might not want you pushing a bunch of changes to it...